bin/geographic_gatherer.py

```diff
@@ -52,15 +52,17 @@ def arg_parse():
                         help="Port to publish the messages on. Default: automatic")
     parser.add_argument("-n", "--nameservers",
                         help=("Connect publisher to given nameservers: "
-                              "'-n localhost -n 123.456.789.0'. Default: localhost"),
+                              "'-n localhost -n 123.456.789.0'. Use '-n false' to disable. "
+                              "Default: localhost."
+                              ),
```

Comment:
I'm wondering if it would be clearer to have a specific flag for this, eg `disable publisher broadcast`?

Reply:
In my opinion we're not disabling broadcasting, but nameserver connection, which has the side-effect of disabling the publisher address broadcasting. I think the cleanest way would be to make "no nameserver connection" the default, and then the user would explicitly add the required nameserver if needed. But this would of course break the backwards compatibility for users who haven't always used `-n localhost` or similar. I'll think about this.